content/docs/command-reference/add.md

```diff
@@ -145,28 +145,31 @@ not.
   [pattern](https://docs.python.org/3/library/glob.html) specified in `targets`.
   Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`
 
-- `--external` - allow `targets` that are outside of the DVC repository. See
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target (instead of using the current working directory).
+  Directories specified in the path will be created by this command. Note that
+  this can be used [with an external cache](#straight-to-cache) to avoid using
+  the disk.
```

Comment:
`the disk` - why disk?

Reply:
`instead of using the current working directory` - not exactly correct. I think it's instead of some default path + default file name?

Reply:
"disk space", "to disk", etc. are common ways to refer to machine storage in general I think. I guess "the disk" not so much haha. Changed.

> instead of some default path + default file name

It's `./basename`, that's what I mean by "the cwd". Changed the sentence to "to place a data target" so it's clearer.

Thanks